tests/unit/slowlog.tcl

```diff
@@ -248,4 +248,37 @@ start_server {tags {"slowlog"} overrides {slowlog-log-slower-than 1000000}} {
         
         $rd close
     }
+
+    foreach is_eval {0 1} {
+        test "SLOWLOG - the commands in script are recorded normally - is_eval: $is_eval" {
+            if {$is_eval == 0} {
+                r function load replace "#!lua name=mylib \n redis.register_function('myfunc', function(KEYS, ARGS) server.call('ping') end)"
+            }
+
+            r client setname test-client
+            r config set slowlog-log-slower-than 0
+            r slowlog reset
+
+            if {$is_eval} {
+                r eval "server.call('ping')" 0
+            } else {
+                r fcall myfunc 0
+            }
+            set slowlog_resp [r slowlog get 2]
```

Comment:
The test would become much more readable if we can store index 0 (eval) and index 1 (functions) in separate variable and do the rest of assertion on these.

Reply:
Also, I think we do **too** much testing with both functions and eval scripts. They both largely flow through the same code base. Just running an EVAL test here is likely sufficient.

Reply:
Ah yes, this test is not very easy to read. I am going the keep the both eval and function since it does not cause harm.